### PR TITLE
Ordering parameter and function created 

### LINKED ISSUE #6: Order items

### PR DESCRIPTION
**Pull request by DanAmador**
Created an order parameter which orders the bookmark bar by title.

resolves #6

### CODE REVIEW

File: lib/bookmarker/cli.ex [hunk 1]
```diff
@@ -16,6 +17,7 @@ defmodule Bookmarker.CLI do
       h: :help,
       f: :file,
       t: :title,
+      or: :order,
```
Comment (lubien) [inline lib/bookmarker/cli.ex]: I think it would be best to leave a letter only version '-o' instead of '-or'.

**Comment by lubien** [comment]
I just merged another PR you might need to rebase your repo.

File: lib/bookmarker/cli.ex (hunk 1 above)
Comment (DanAmador) [reply lib/bookmarker/cli.ex]: I chose the use or as the char "o" was already being used for the "output" key  in line 24.

Comment (lubien) [reply lib/bookmarker/cli.ex]: Oh. My bad.

Maybe we could rename the flag to "--sort", "-s".

Comment (DanAmador) [reply lib/bookmarker/cli.ex]: Great! I'll fix that in a sec :+1:

File: lib/bookmarker/cli.ex [hunk 2]
```diff
@@ -81,7 +81,7 @@ defmodule Bookmarker.CLI do
                             Default: Google Chrome Bookmarks
       -d, --description     Set a description for the rendered markdown.
                             Default: Generated by Bookmarker
-      -or, --order          Set if the list is ordered
+      -s, --s          Set if the list is ordered
```
Comment (lubien) [inline lib/bookmarker/cli.ex]: Don't forget the full word "--sort".

image

Also don't forget to indent

File: lib/bookmarker/cli.ex [hunk 3]
```diff
@@ -43,58 +45,61 @@ defmodule Bookmarker.CLI do
...
-                            Default: Google Chrome Bookmarks
-      -d, --description     Set a description for the rendered markdown.
-                            Default: Generated by Bookmarker
-      --no-timestamp        Prevent appending of build datetime after description.
-                            Default: use timestamp
-      -i, --ignore          Ignore folders. You may set this multiple times.
-      -p, --path            Restrict to folder. You may set this one time only.
-      -o, --output          Save rendered markdown to a file.
-                            Default: none (output to stdin)
+    -f, --file            Set where your chrome bookmarks file is.
```
Comment (lubien) [inline lib/bookmarker/cli.ex]: Sorry but you broke the original indentation

Comment (lubien) [reply lib/bookmarker/cli.ex]: Please use the previous format.

**Pull request closed without merging**